test/test-runner.env

```diff
@@ -0,0 +1,8 @@
+# You can override any of these in ../../local.env
+HEADED_TESTS=false
+MOCHA_OPTS_TIMEOUT=120000
```

Comment:
This is an increase from the previous default of 90 seconds which was often not enough in CI to get through the Example test suite. WIth some additional elaboration we could set a different timeout for CI vs local, but this extra 30 seconds of timeout hasn't hurt me locally.